core/src/main/resources/error/error-classes.json

```
@@ -64,6 +64,16 @@
       }
     }
   },
+  "AVRO_INCORRECT_TYPE" : {
+    "message" : [
+      "Cannot convert Avro <avroPath> to SQL <sqlPath> because the original encoded data type is <avroType>, however you're trying to read the field as <sqlType>, which would lead to an incorrect answer. To allow reading this field, enable the SQL configuration: `<key>`."
```

You have already quoted `key` by `toSQLConf`, please, remove quotes here:
```suggestion
      "Cannot convert Avro <avroPath> to SQL <sqlPath> because the original encoded data type is <avroType>, however you're trying to read the field as <sqlType>, which would lead to an incorrect answer. To allow reading this field, enable the SQL configuration: <key>."
```

---

sg!